tests/unit/core/reducers/test_languageTools.js

```diff
@@ -77,7 +80,7 @@ describe(__filename, () => {
     it('returns an array of languages', () => {
       const lang = 'fr';
       const language = createFakeLanguageTool({ target_locale: lang });
```

Comment:
I suggest we change the default in `createFakeLanguageTool()`:

```diff
diff --git a/tests/unit/helpers.js b/tests/unit/helpers.js
index 904dfe480..9331b75f0 100644
--- a/tests/unit/helpers.js
+++ b/tests/unit/helpers.js
@@ -954,7 +954,7 @@ export function createFakeLanguageTool({
     default_locale: DEFAULT_LANG_IN_TESTS,
     guid: fakeAddon.guid,
     locale_disambiguation: '',
-    name: createLocalizedString(name, target_locale),
+    name: createLocalizedString(name, 'en-US'),
     target_locale,
     type: ADDON_TYPE_LANG,
     url: 'https://addons.allizom.org/en-US/firefox/addon/acholi-ug-lp-test',
```

This will avoid issues in the future. We don't really care about `target_locale` really.

Reply:
instead of hardcoding `en-US`, we can use `DEFAULT_LANG_IN_TEST`